pyproject.toml

```diff
@@ -104,20 +104,20 @@ docs = [
 ]
 optional = [
     'SimpleITK',
-    'astropy>=3.1.2',
+    'astropy>=5.0',
     'cloudpickle>=0.2.1',
-    'dask[array]>=1.0.0,!=2.17.0',
-    'matplotlib>=3.3',
-    'pooch>=1.3.0',
+    'dask[array]>=2021.1.0',
+    'matplotlib>=3.5',
+    'pooch>=1.6.0',
     'pyamg',
 ]
 test = [
     'asv',
     'codecov',
-    'matplotlib>=3.3',
-    'pooch>=1.3.0',
-    'pytest>=5.2.0',
-    'pytest-cov>=2.7.0',
+    'matplotlib>=3.5',
```

Comment:
Also happy to keep this at 3.6 and update the docs; up to you.

Reply:
Let's keep it at 3.5 for now and go to 3.6 in 0.22.